odd-platform-ui/src/components/DataQuality/DataQualityFilters/DataQualityFiltersStyles.ts

```diff
@@ -0,0 +1,15 @@
+import { Grid } from '@mui/material';
+import styled from 'styled-components';
+
+export const Container = styled('div')(({ theme }) => ({
+  padding: theme.spacing(2, 1, 1.5, 1),
+}));
+
+export const FacetsLoaderContainer = styled(Grid)(({ theme }) => ({
+  padding: theme.spacing(0, 3),
+  justifyContent: 'center',
+}));
```

Comment:
I don't mind but I have a controversial feelings about mui/material. I know it's a lot of code with this mixed approach here but I think if we have the styled-components so lets use only them. Because it feels redundant otherwise. For example this component can be just this:
```
<Grid justifyContent='center' px={3}>

or

const FacetsLoaderContainer = styled('div')(
  ({ theme }) => css`
    display: 'flex'
    justifyContent: 'center'
    padding: ${theme.spacing(0)} ${theme.spacing(3)};
  `
);
```

What do you think?

Reply:
using styled-components for new components sounds good